awesome-generator/awesome.toml

```
@@ -258,6 +258,7 @@
 "https://gitlab.com/ravenfeld/Connect-IQ-DataField-Speed" = { description = "Colorized speed gauge" }
 "https://gitlab.com/ravenfeld/Connect-IQ-DataField-VirtualPartner" = { description = "Virtual race partner" }
 "https://gitlab.com/ravenfeld/connect-iq-datafield-runner" = { description = "Fields of data that can correct the distance course by pressing lap" }
+"https://github.com/ray0711/BTHomeIQ" = { description = "ConnectIQ datafield for displaying and recording BTHome temperature broadcasts" }
```

Thanks! The repo seems to have a description so we don't need to add it. It's better to leave it out if the description of the repo changes, that way it will be updated in this list automatically.

```suggestion
"https://github.com/ray0711/BTHomeIQ" = {}
```